includes/data.php

```diff
@@ -1950,18 +1950,35 @@ function pods_serial_comma( $value, $field = null, $fields = null, $and = null,
 			$format = pods_v( 'repeatable_format', $params->field, 'default', true );
 
 			if ( 'default' !== $format ) {
-				$params->serial = false;
 
-				if ( 'custom' === $format ) {
-					$separator = pods_v( 'repeatable_format_separator', $params->field );
+				switch ( $format ) {
+					case 'ul':
+					case 'ol':
+						if ( is_array( $value ) ) {
+							$value = '<' . $format . '><li>' . implode( '</li><li>', $value ) . '</li></' . $format . '>';
+						}
+					break;
+					case 'br':
+						if ( is_array( $value ) ) {
+							$value = implode( '<br />', $value );
```

Comment:
Should we add `\n` breaks before or after the br tags for HTML readability?

Reply:
Personally I'm not fond of this, then again, since <br> tags are inline it shouldn't really matter.
White spaces are and `\n` tags are very annoying when it comes to inline-block elements.